api/v1alpha1/performanceprofile_conversion.go

```diff
@@ -0,0 +1,120 @@
+package v1alpha1
+
+import (
+	"github.com/openshift-kni/performance-addon-operators/api/v1"
+	"sigs.k8s.io/controller-runtime/pkg/conversion"
+)
+
+// ConvertTo converts this PerformanceProfile to the Hub version (v1).
+func (curr *PerformanceProfile) ConvertTo(dstRaw conversion.Hub) error {
+	dst := dstRaw.(*v1.PerformanceProfile)
+
+	// ObjectMeta
+	dst.ObjectMeta = curr.ObjectMeta
+
+	// Spec
+	dst.Spec.CPU = new(v1.CPU)
+	hugePagesSize := v1.HugePageSize(*curr.Spec.HugePages.DefaultHugePagesSize)
+	reserved := v1.CPUSet(*curr.Spec.CPU.Reserved)
+	dst.Spec.CPU.Reserved = &reserved
+	isolated := v1.CPUSet(*curr.Spec.CPU.Isolated)
+	dst.Spec.CPU.Isolated = &isolated
+	dst.Spec.CPU.BalanceIsolated = curr.Spec.CPU.BalanceIsolated
+
+	dst.Spec.HugePages = new(v1.HugePages)
+	dst.Spec.HugePages.DefaultHugePagesSize = &hugePagesSize
+	dst.Spec.HugePages.Pages = make([]v1.HugePage, len(curr.Spec.HugePages.Pages))
+	for i, p := range curr.Spec.HugePages.Pages {
+		dst.Spec.HugePages.Pages[i] = v1.HugePage{
+			Size: v1.HugePageSize(p.Size), Count: p.Count, Node: p.Node,
```

Comment:
`Node: p.Node` will assign pointers
so probably you should
```
if p.Node != nil {
    dst.Spec.HugePages.Pages[i].Node = pointer.Int32Ptr(*p.Node)
}
```

Reply:
done